### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by ishimwezachee**
- An API for the project Covid
- Followed design guidelines
- Interactions:
- Home page: When the page loads, the web app shows the Country's name, TOTAL-CONFIRMED TODAY RECOVERED and ------ When the user clicks (or taps) on a MORE INFO the application navigates to the details page.
- Details page: the web app retrieves data from the API to show detailed data in the MORE INFO and When the user clicks on --- the "Back" button (<), the user navigates to the home page.
- Created unit and integration tests.
- The project is a single-page application (SPA) built with React and Redux.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project is deployed and accessible online Live Link and also (added a link to the online version of the app to README file).
- Recorded a video for documentation Loom Record

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @ishimwezachee 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,70 +1,58 @@
...
+[LINK Live](https://quirky-austin-ee8614.netlify.app/)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Vidoe Description
 
-### `npm run build`
+[Video Demo](https://www.loom.com/share/c1bafc0ccb0d4ed4995be48480abd796)
```
Comment (mikenath223) [inline README.md]: - [ ] In the video you are expected to talk about a piece of code that you found interesting. Please record an additional video highlighting this.

File: src/App.test.js [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+const APP = (
+  <Provider store={store}>
+    <App />
+  </Provider>
+);
+
+describe('App Renders test', () => {
+  it('Renders App', () => {
+    const app = renderer.create(APP).toJSON();
+    expect(app).toMatchSnapshot();
```
Comment (mikenath223) [inline src/App.test.js:28]: - [ ] Some snapshot tests are failing kindly update them.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- All tests pass now
- You showed an interesting part of your code.
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] I suggest that you consider adding instructions on how to clone and run your project to your README file. This would ensure that anyone, no matter their programming background would be able to run your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @ishimwezachee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**